src/monster.cpp

```diff
@@ -507,10 +507,17 @@ void monster::try_reproduce()
         }
 
         chance += 2;
+
+        bool friendly_parent = false;
+        // wildlife creatures that are friendly to the player will spawn friendly offspring
+        if( type->in_category( "WILDLIFE" ) &&
+            attitude_to( get_player_character() ) == Attitude::A_FRIENDLY ) {
+            friendly_parent = true;
+        }
```

Comment:
```suggestion
        // wildlife creatures that are friendly to the player will spawn friendly offspring
        const bool friendly_parent = type->in_category( "WILDLIFE" ) &&
                                     attitude_to( get_player_character() ) == Attitude::A_FRIENDLY;

```

using `const` is preferred over mutating values as it helps reduce bugs and makes it easier to follow code (less moving parts).